kedro/templates/project/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/pipeline_registry.py

```diff
@@ -1,13 +1,20 @@
 """Project pipelines."""
 from typing import Dict
 
+from kedro.framework.project import find_pipelines
 from kedro.pipeline import Pipeline, pipeline
 
 
 def register_pipelines() -> Dict[str, Pipeline]:
     """Register the project's pipelines.
 
+    Since Kedro 0.18.3, projects can use the ``find_pipelines`` function
+    to autodiscover pipelines. However, projects that require more fine-
+    grained control can still construct the pipeline mapping without it.
```

Comment:
I think this will probably need a bit more explanation (e.g. what structure is required for a pipeline to be auto-discovered, which is that created by `kedro pipeline create`). Or maybe just a link to the docs once it's in there.

Reply:
@AntonyMilneQB I've made a new ticket to add pipeline registry documentation (starting with the basic documentation that doesn't currently exist), so I'll resolve this for now.